tensorflow/tools/dist_test/scripts/k8s_tensorflow.py

```diff
@@ -190,15 +232,19 @@ def main():
     sys.exit(1)
 
   # Generate contents of yaml config
-  yaml_config = GenerateConfig(args.num_workers,
+  yaml_config = GenerateConfig(args.cluster_name,
+                               args.creator,
+                               args.num_workers,
                                args.num_parameter_servers,
                                args.grpc_port,
                                args.request_load_balancer,
                                args.docker_image)
   print(yaml_config)  # pylint: disable=superfluous-parens
 
 
-def GenerateConfig(num_workers,
+def GenerateConfig(cluster_name,
```

Comment:
Can you make this change backward compatible by letting cluster_name and creator be keyward arguments that have None values? So if some user doesn't want to explicitly set the arguments, they can just ignore them, and the pods, rcs and services will have the same naming pattern as before, i.e., without the prefixes like "cluster1-"?